runtimes/pythonrt/build.go

```diff
@@ -58,8 +63,71 @@ func (py *pySvc) Build(ctx context.Context, fsys fs.FS, path string, values []sd
 	var art sdktypes.BuildArtifact
 	art = art.WithCompiledData(compiledData)
 
-	// TODO: We don't have exports for now, UI can change singleshot to a text
-	// box instead of dropdown
+	exports, err := findExports(fsys)
+	if err == nil {
+		art = art.WithExports(exports)
+	} else {
+		py.log.Warn("can't get exports", zap.Error(err))
+
+	}
 
 	return art, nil
 }
+
+func findExports(fsys fs.FS) ([]sdktypes.BuildExport, error) {
+	codeDir, err := os.MkdirTemp("", "ak-proj")
+	if err != nil {
+		return nil, err
+	}
+
+	if err := os.CopyFS(codeDir, fsys); err != nil {
+		return nil, err
+	}
+
+	runnerDir, err := os.MkdirTemp("", "ak-runner")
```

Comment:
oh wow it's actually running the runner for this?

Reply:
Yes.